app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -136,9 +183,4 @@ export type APIProjectType = {
   +numberOfOpenAssignments: number,
 };
 
-export type APIScriptType = {
-  +id: string,
-  +name: string,
-  +owner: APIUserType,
-  +gist: string,
-};
+export default {};
```

Comment:
There'll be a linting error otherwise ` error  Prefer default export  import/prefer-default-export` as there is only one other export (that is not a flow type!) and the linter would like that to be made the default export then, which I don't think would be fitting.